<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KDebugDialog</class>
 <widget class="QWidget" name="KDebugDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" rowspan="5">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="KTreeWidgetSearchLineWidget" name="m_incrSearch"/>
     </item>
     <item>
      <widget class="QTreeWidget" name="m_areaWidget">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="rootIsDecorated">
        <bool>false</bool>
       </property>
       <property name="uniformRowHeights">
        <bool>true</bool>
       </property>
       <property name="itemsExpandable">
        <bool>false</bool>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="pInfoGroup">
     <property name="title">
      <string>Information</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Output to:</string>
        </property>
        <property name="buddy">
         <cstring>pInfoCombo</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="KComboBox" name="pInfoCombo"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Filename:</string>
        </property>
        <property name="buddy">
         <cstring>pInfoFile</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="KLineEdit" name="pInfoFile"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QGroupBox" name="pWarnGroup">
     <property name="title">
      <string>Warning</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Output to:</string>
        </property>
        <property name="buddy">
         <cstring>pWarnCombo</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="KComboBox" name="pWarnCombo"/>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Filename:</string>
        </property>
        <property name="buddy">
         <cstring>pWarnFile</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="KLineEdit" name="pWarnFile"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QGroupBox" name="pErrorGroup">
     <property name="title">
      <string>Error</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Output to:</string>
        </property>
        <property name="buddy">
         <cstring>pErrorCombo</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="KComboBox" name="pErrorCombo"/>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Filename:</string>
        </property>
        <property name="buddy">
         <cstring>pErrorFile</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="KLineEdit" name="pErrorFile"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QGroupBox" name="pFatalGroup">
     <property name="title">
      <string>Fatal Error</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Output to:</string>
        </property>
        <property name="buddy">
         <cstring>pFatalCombo</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="KComboBox" name="pFatalCombo"/>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Filename:</string>
        </property>
        <property name="buddy">
         <cstring>pFatalFile</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="KLineEdit" name="pFatalFile"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="pAbortFatal">
     <property name="text">
      <string>Abort on fatal errors</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QCheckBox" name="m_disableAll2">
     <property name="text">
      <string>Disable all debug output</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KTreeWidgetSearchLineWidget</class>
   <extends>QWidget</extends>
   <header>ktreewidgetsearchline.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
